run_combination_test.py

```diff
@@ -71,6 +71,7 @@ def get_shuffle_params(params, index):
     if params['h5py'] == '2.6':
         conf['requires'].append('h5py<2.7')
     elif params['h5py'] == '2.5':
+        conf['requires'].append('numpy<1.10')
```

Comment:
Current chainer-test uses `pip install -U`, so the following code updates numpy:

```sh
pip install -U 'numpy<1.10'
pip install -U 'h5py<2.6'  # updates numpy
```